configuration/webpack.config.prod.js

```diff
@@ -152,7 +152,11 @@ module.exports = {
             // Pending futher investigation:
             // https://github.com/terser-js/terser/issues/120
             inline: 2,
-            drop_console: true,
+            pure_funcs: [
```

Comment:
Why do we need this change? There is a separate logger in this package that should be used instead anyways

Reply:
I got this change from `non-prod` webpack config https://code.amazon.com/packages/AmazonConnectChatInterface/blobs/bcd1de70812fde41d1c47bdbbaeac19580fe0b82/--/configuration/webpack.config.prod.js#L157
Without this change - currently the application does not load when I tested my changes in https://github.com/amazon-connect/amazon-connect-chat-ui-examples/ -- Root Cause: console.info/console.log are overridden to `undefined` which breaks/errors during app rendering.